modules/microshift-config-yaml.adoc

```diff
@@ -4,81 +4,43 @@
 
 :_content-type: CONCEPT
 [id="microshift-config-yaml_{context}"]
-= Using a YAML configuration file
+= Configuring {product-title}
 
-Configure your {product-title} using a YAML file. The values specified in your configuration file are used when you execute a command.
-// Q: can these values be overriden with CLI flags?
+{product-title} searches for a configuration file in the user-specific directory, `~/.microshift/config.yaml`, then the system-wide `/etc/microshift/config.yaml` directory. Create the configuring file, specifying any settings that should override the defaults, before starting MicroShift.
 
 [id="microshift-yaml-default_{context}"]
 == Default settings
 If you do not create a `config.yaml` file, the default values are used.
-// Q: what creates the default yaml?
-// Q: can default file be modified? (instead of making a new one)
```

Comment:
You only have to include the settings you want to change in the config file, but you have to put them in a file.